assets/default-config.yaml

```diff
@@ -17,13 +17,20 @@
 #
 
 filters:
-  - namespace: istio-system
-    exporters:
+  - reason: ""     # filter events of the specified reason, regular expression like "Killing|Killed" is supported.
```

Comment:
Should we set some recommended reasons and other configurations? Such as for mesh case?

Reply:
> Should we set some recommended reasons and other configurations? Such as for mesh case?

I haven't come up with some recommended reasons and other configurations for now, maybe I can fine-tune these when applying them into real use case later

Reply:
OK. I read you are adding documentation, I thought you are going to release this.